doc/nrf/releases_and_maturity/migration/migration_guide_2.6.rst

```diff
@@ -2,797 +2,862 @@
 
 .. _migration_2.6:
 
-Migration guide for |NCS| v2.6.0 (Working draft)
-################################################
+Migration guide for |NCS| v2.6.0
```

Comment:
I don't know the layout, but it could be useful to point to the zcbor migrations notes https://github.com/zephyrproject-rtos/zephyr/blob/main/doc/releases/migration-guide-3.6.rst#zcbor

Reply:
@carlescufi I did the same changes as 'fromtree' in sdk-zephyr, so it's all relevant.

Reply:
Added the content to Migration guide.